### DISCUSSION TITLE
Snapshots partially cut off

### DISCUSSION THREAD

**Discussion by oreowhiskers** [icosa-gallery/open-brush#482] [General]
Hi, I saw there was another bug that had been fixed about snapshots being fully black, but I've had issues with snapshots being partially or almost even half consumed by black. It seems very random, and it's really frustrating to have to take a bunch of snapshots just to hope one turns out ok! I've attached some pictures of the snapshots to show off the effect. I'm currently on the Meta Quest 2 and I have experimental features turned off, let me know if it would help to give any more information!

![image](https://github.com/icosa-gallery/open-brush/assets/86029945/4763086e-b2fd-4a71-a76b-d9f613f05832)
![image](https://github.com/icosa-gallery/open-brush/assets/86029945/2748595b-576a-4055-85a4-cc449079150c)
![image](https://github.com/icosa-gallery/open-brush/assets/86029945/32cbf528-97de-456b-9694-e3a2e0b519e1)
![image](https://github.com/icosa-gallery/open-brush/assets/86029945/74d91197-ddbf-4c5b-a2b9-6b2d9b78f529)

**Comment by mikeskydev** [comment]
Hi, apologies for the late response. This has now been fixed and is currently available on our Beta release, which should become the main realease very shortly!
